src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -64,6 +64,7 @@ void searchByInstances_parameterized_singleResult(String query, String value) th
     "uniformTitle all {value}, deja vu",
     "uniformTitle all {value}, déjà vu",
     "contributors.name all {value}, franks",
+    "contributors.authorityId == {value}, 11110000-fcf6-45cc-b6da-4420a61ef72c",
```

Comment:
Please add a test for the alias
`authorityId == {value}`

Reply:
done